lib/assessment.js

```diff
@@ -4,32 +4,60 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
-
+var min = function min(nums) {
+  var minValue = nums[0];
+  for (var i = 1; i < nums.length; i++) {
+    if (nums[i] < minValue) {
+      minValue = nums[i];
+    }
+  }
+  return minValue;
```

Comment:
This would work great if an array of numbers were passed in as the only argument. If you replace all mentions of `nums` with `arguments` (and empty the parameter list), this works as intended. 3/4